alacritty/src/cli.rs

```diff
@@ -297,6 +304,19 @@ pub struct WindowOptions {
     pub window_identity: WindowIdentity,
 }
 
+/// Parameters to the `config` IPC subcommand.
+#[cfg(unix)]
+#[derive(Args, Serialize, Deserialize, Default, Debug, Clone, PartialEq, Eq)]
+pub struct IpcConfig {
+    /// Configuration file options [example: cursor.style=Beam].
+    #[clap(required = true, value_name = "CONFIG_OPTIONS")]
+    pub options: Vec<String>,
+
+    /// Window ID for the new config
+    #[clap(long, short)]
+    pub window_id: Option<u64>,
```

Comment:
Could you get the default out of the `env` variable, so you don't have to specify it when sending message? I think that's how most applications do work that way.

Would probably need to have an option to explicitly target all, like `"*"`.

Reply:
I'm not sure if the most likely usecase is to change all terminals, or just one? Based on that I would choose the default. I've chosen changing all here, but I'm not sure.

Reply:
I think just one? If you want all you change the config most of the time? We could have an alias to target all though?

Reply:
Yeah the main problem is around providing a sensible and nicely documented way to do it for all windows. I'm not really a fan of `"*"` or a dedicated flag, but I don't have any better ideas either.